test/index.js

```diff
@@ -59,4 +59,8 @@ describe('XML formatter', function () {
         assertFormat('test/data7_white-space-on-closing-tag/xml*-input.xml', {whiteSpaceAtEndOfSelfclosingTag: true}, done);
     });
 
+    it('should format XML without indentation', function(done) {
+        assertFormat('test/data9/xml*-input.xml', {indentation: ''}, done);
```

Comment:
It seems that what you are after is linearisation of the XML which I think should be done by a combination of `indentation: ''` and `lineSeparator: ''`.

If `indentation: ''` only, we should still have the XML formatted in multiple lines with the `lineSeparator` option honoured.

Reply:
@chrisbottin Thanks for pointing that out, you're right. I'll change it to format without indentation or line separator when given the option `{indentation: '', lineSeparator: ''}`.